core/trino-main/src/test/java/io/trino/block/AbstractTestBlock.java

```diff
@@ -25,6 +25,7 @@
 import io.trino.spi.block.DictionaryBlock;
```

Comment:
commit message is too long (should be <= 70). You can use commit descriptions below title